plc4j/spi/src/main/java/org/apache/plc4x/java/spi/values/PlcValueHandler.java

```diff
@@ -83,6 +83,9 @@ public static PlcValue of(Object[] values) {
         if (value instanceof Long) {
             return PlcLINT.of(value);
         }
+        if (value instanceof BigInteger) {
+            return PlcDINT.of(value);
```

Comment:
Woudln't a LINT be the better match, as a BigInteger could actually be even bigger than the DINT value space?

Reply:
Yes, PlcLINT is better choose.